From eda98fcdd28ea98fb21ee6e7641c045db8ded334 Mon Sep 17 00:00:00 2001
From: SivaPriyaSanda459 <sivapriya.sanda@votarytech.com>
Date: Wed, 10 Jun 2026 13:08:22 +0530
Subject: [PATCH 2/2] changed in file1

---
 file1.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/file1.txt b/file1.txt
index 9f4d96d..0df57e6 100644
--- a/file1.txt
+++ b/file1.txt
@@ -1 +1,2 @@
 Hello Git
+hi iam siva Priya
-- 
2.25.1

